responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -7,6 +7,7 @@ channels:
   - pytorch
 dependencies:
   - python=3.8
+  - pip
```

Comment:
nit: i'd recommend to pin pip

Reply:
[Task 2202691](https://msdata.visualstudio.com/Vienna/_workitems/edit/2202691): curated image update for findlink, pip